src/components/Search.js

```diff
@@ -0,0 +1,19 @@
+import React, {useState} from 'react'
+import {connect} from 'react-redux';
+import {searchPlayer} from '../actions/';
+
+const Search = (props) => {
+    const [player, setPlayer] = useState("");
+    console.log(player);
+    return (
+        <div>
+            <input 
+                type="text"
+                value={player.name}
+                placeholder="Search for a Player"
+                onChange={(e) => setPlayer(e.target.value)}></input>
+            <button onClick={() => props.searchPlayer(player)}>Search Player</button>
+        </div>
+    )
+}
+ export default connect(null, {searchPlayer: searchPlayer})(Search);
```

Comment:
Just gonna point it out once but, similar to destructing props, when setting key value pairs, if the key name is the same as the value name you can do `{ searchPlayer }` instead of `{ searhPlayer: searchPlayer }`